conf/routes

```diff
@@ -94,7 +94,9 @@ GET  /$country<(uk|us|au|eu|int|nz|ca)>/subscribe/digital     controllers.Digita
 GET  /$country<(uk|us|au|eu|int|nz|ca)>/subscribe/digital/checkout  controllers.Application.permanentRedirectWithCountry(country, location="/subscribe/digital/checkout")
 GET  /subscribe/digital/checkout  controllers.DigitalSubscription.displayForm()
 GET  /subscribe/digital/checkout/thankyou-existing  controllers.DigitalSubscription.displayThankYouExisting()
-POST /subscribe/digital/create  controllers.DigitalSubscription.create
+
+# Deprecated, soon all requests to create a subscription should go via /subscribe/create instead
```

Comment:
I guess we should leave this here for a short while because some users might have the old JS when this is first deployed(?) and I don't want them to get 404s if they try to pay...